tools/release.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+'use strict';
+
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+const { buildDockerImages, pushDockerImage } = require('./docker-utils');
+
+async function main() {
+  const argv = yargs(hideBin(process.argv))
+    .option('defaultNodeDockerVersion', {
+      alias: 'default-node-docker-version',
```

Comment:
I strongly dislike having to use `--camelCasedNames` on the command line, so I figured I might as well allow both camel and kebab case

Reply:
I don't really mind either way, now if you started debating spaces vs tabs, we'd have to have a serious conversation :sarcasm:

Reply:
:smile: